scripts/compare-fscontent/compare-fscontent.go

```diff
@@ -63,20 +63,29 @@ func getFSContent(c *rpcclient.Client) ([][]byte, []*netmap.NetmapNode, error) {
 	return containers, netmap, nil
 }
 
-func cliMain(c *cli.Context) error {
-	a := c.Args().Get(0)
-	b := c.Args().Get(1)
-	if a == "" {
-		return errors.New("no arguments given")
-	}
-	if b == "" {
-		return errors.New("missing second argument")
+var (
+	compareFsContentFirstNodeAddress  string
+	compareFsContentSecondNodeAddress string
+	compareFsContentIgnoreHeightFlag  bool
```

Comment:
Likely `compareFsContent` prefix can be dropped, there is nothing else there in this package.